xray/src/generation.rs

```diff
@@ -471,37 +471,51 @@ pub struct Tile {
     pub resolution: f64,
 }
 
+pub struct XrayParameters {
+    pub point_cloud_client: PointCloudClient,
+    pub query_from_global: Option<Isometry3<f64>>,
+    pub filter: HashMap<String, ClosedInterval<f64>>,
```

Comment:
Nit: `filter` sounds like a verb, maybe something like `filter_intervals`?
Also, why are these fields specifically XrayParameters but not `image_size` or `coloring_strategy`?

Reply:
I renamed it.
They are shared between `xray_from_points` and `build_xray_quadtree`, but `image_size`/`coloring_strategy` are not.